kingpin/actors/aws/s3.py

```diff
@@ -378,6 +378,46 @@ class LifecycleConfig(SchemaCompareBase):
     }
 
 
+class NotificationConfiguration(SchemaCompareBase):
+
+    """Provides JSON-Schema based validation of the supplied Notification Config.
+
+   .. code-block:: json
+
+      {
+         "queueconfigurations": [
```

Comment:
Shouldn't this be in snake_case so that it properly gets converted to camelcase when making the API call to AWS?

Reply:
done